ts/store/middlewares/analytics.ts

```diff
@@ -141,6 +144,13 @@ const trackAction = (mp: NonNullable<typeof mixpanel>) => (
     case getType(profileFirstLogin):
       return mp.track(action.type, action.payload);
 
+    // dispatch to mixpanel when the email is validated
+    case getType(emailValidationChanged):
```

Comment:
You don't need another const to describe the action type, it's already in the action

```typescript
case getType(emailValidationChanged):
      return mp.track(action.type, action.payload);
```
This handling looks like the same of others (ex **profileFirstLogin**) so they could be coupled

Reply:
If we choose to represent the action as `emailValidated` instead of `emailValidationChanged` the actual check isn't needed anymore and can be changed like a standard handler.

Reply:
Yes it's true.
In both case 
`const emailValidationComplete = "EMAIL_VALIDATION_COMPLETED";`
It's not needed because

`action = {type: "EMAIL_VALIDATION_COMPLETED", payload: true/false}`